### ISSUE TITLE
[Anti-adblock] mail.yahoo.com - "wrong" filter

### ISSUE THREAD

**Issue by Procyon-b** [uBlockOrigin/uAssets#8038]
### URL(s) where the issue occurs
`https://mail.yahoo.com/`

### Describe the issue

related to this [issue](https://github.com/NanoAdblocker/NanoFilters/issues/501)
which created this filter `mail.yahoo.com##:xpath(//*[contains(text(),"ad-blocker")]):upward(3)` in "uBlock filters" ; @mapx- 

That filter has 2 problems:
1. it filters only when the language of the page is in english, but doesn't if yahoo's interface is localized
2. if you paste the string `ad-blocker` in a new message, the content of the message disappears. If you write it its ok until you hit "enter". Opening the draft afterwards will trigger the filter.
if you open a message containing this text, it triggers the filter and the content disappears

### Versions

- Browser/version: chrome
- uBlock Origin version: 1.30.4

### Notes

You can change yahoo's language by appending `.lang=fr-BE` to the url. You'll get it in french:
try `https://mail.yahoo.com/d/folders/1?.lang=fr-BE` to verify "1."

To trigger the anti-adblock warning you have to wait a minute while the inbox is being displayed. You'll notice at first a blank line between the header "**Today**" (if you have received mail today) ("**Aujourd'hui**" with the test in french) and the first message. It will disappear after about a minute. Opening a message then triggers the anti ad-block message.

I can suggest this filter as a replacement:
`mail.yahoo.com##div#modal-outer[aria-labelledby="adblock-unblock-cue"]`
It works with all the languages I have tested, and doesn't have problem "2."

**Comment by krystian3w** [comment]
> You can change yahoo's language by appending .lang=fr-BE to the url. You'll get it in french:
try `https://mail.yahoo.com/d/folders/1?.lang=fr-BE` to verify "1."

IMO no always works, my account enforce Polish 🇫🇷  instead French 🇫🇷 with Belgium 🇧🇪  changes.

**Comment by Procyon-b** [comment]
@krystian3w I have tested with `https://mail.yahoo.com/d/folders/1?.intl=pl&.lang=pl-PL` and it works. Do you have another behaviour ?

**Comment by krystian3w** [comment]
Yes on `https`.

https://github.com/AdguardTeam/AdguardFilters/issues/61441#issuecomment-680030330

**Comment by PIZZA90** [comment]
I see an ad unit above, in a search of **yahoo**  

![Captura](https://user-images.githubusercontent.com/33556647/96647437-f16f6e80-132d-11eb-9f3d-575889aebe23.PNG)

searching in github... i found a similar filter 🤔 

https://github.com/troysjanda/MyBlockLists/blob/d9451c1ab67eda80190738ac170416aa7922ca39/yahoocomplete.txt#L141

**Comment by krystian3w** [comment]
But `www.yahoo.com` =/= `mail.yahoo.com`

Open new issue with filed template.

**Comment by PIZZA90** [comment]
reported in easylist..

what about **easylist** they don't have much activity in the repository? 🙄

**Comment by Procyon-b** [comment]
Searching on `es.search.yahoo.com` for the word `decathlon` brings a result page with this ad.
The ad container can be targeted by the 1st filter below
```
search.yahoo.com##li.first>.sys_shopleguide.dd
search.yahoo.com##ol.searchCenterTopAds
```
The 2nd filter is to get rid of the "Anuncios..." line (also present on pages where no ad is shown).

**Comment by mapx-** [comment]
`search.yahoo.com##.sys_shopleguide` is working fine
tested for the italian search too (search for "bici elettriche"), same filter

@ryanbr

**Comment by ryanbr** [comment]
> what about easylist they don't have much activity in the repository? 🙄

Wat.

**Comment by PIZZA90** [comment]
about the comment of **krystian3w**

I understood, that after a month without solution in easylist repository, uasset will take care of the issues, my bad english and the google traductor sucks.. 🤗 

I did not understand that comment very well, was a misunderstanding.

all ok 😅
